assets/src/dashboard/app/views/shared/storyListView.js

```diff
@@ -92,13 +97,33 @@ const toggleSortLookup = {
   [SORT_DIRECTION.ASC]: SORT_DIRECTION.DESC,
 };
 
+const LastModifiedTableHeaderCell = styled(TableHeaderCell)`
+  min-width: 160px;
+`;
+
+const AuthorTableHeaderCell = styled(TableHeaderCell)`
+  min-width: 110px;
+`;
+
 export default function StoryListView({
   filteredStories,
   storySort,
   handleSortChange,
   handleSortDirectionChange,
   sortDirection,
+  tags,
+  categories,
+  users,
 }) {
+  const metadataStringForIds = useCallback((metadata, ids) => {
+    const metadataString = ids
+      .reduce((memo, current) => {
+        return [...memo, metadata[current].name];
+      }, [])
+      .join(', ');
```

Comment:
To test this, I went through my local wordpress set up and added a handful of tags and categories. This reduce is getting caught by having an id that doesn't exist in the metadata and searching for its name. A story of mine has 3 tags, but the tags returned through the api is missing one of those Ids and causing the callback to error out on the reduce.  This is my data, if it helps - but I think just doing a check that the name exists would be good. 

From stories object: 
```
{
22:
author: 1
bottomTargetAction: "http://localhost:8899/wp-admin/post.php?action=edit&post=22"
categories: (2) [9, 11]
centerTargetAction: ""
id: 22
modified: Moment {_isAMomentObject: true, _i: "2020-05-01T15:48:27", _f: "YYYY-MM-DDTHH:mm:ss", _isUTC: false, _pf: {…}, …}
originalStoryData: {id: 22, date: "2020-04-06T21:14:58", date_gmt: "2020-04-06T21:14:58", guid: {…}, modified: "2020-05-01T15:48:27", …}
pages: [{…}]
status: "draft"
tags: (3) [28, 29, 30]
title: "DERPIN WITH FRED"
}
```
Tags returned: 
<img width="980" alt="Screen Shot 2020-05-01 at 9 13 06 AM" src="https://user-images.githubusercontent.com/10720454/80820571-fd068780-8b8b-11ea-8be1-1854bc1012a6.png">

Reply:
Yeah, perhaps this could be fixed with a `?` and `Boolean` filter like so:
```
.reduce((memo, current) => {
  return [...memo, metadata[current]?.name];
}, [])
.filter(Boolean)
.join(', ');
```